Framework/Algorithms/inc/MantidAlgorithms/ChangePulsetime2.h

```diff
@@ -0,0 +1,42 @@
+#ifndef MANTID_ALGORITHMS_CHANGEPULSETIME2_H_
+#define MANTID_ALGORITHMS_CHANGEPULSETIME2_H_
+
+#include "MantidAPI/Algorithm.h"
+#include "MantidKernel/System.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** ChangePulsetime : TODO: DESCRIPTION
+ *
+ * @author
+ * @date 2011-03-31 09:31:55.674594
+ */
+class DLLExport ChangePulsetime2 : public API::Algorithm {
```

Comment:
I suppose this is only a temporary addition and will be removed before merging? Or should we have this as version 2 of the algorithm?